eip155/caip10.md

```diff
@@ -32,11 +32,11 @@ Note that a given address cannot be assumed to work on all current and future ne
 
 ## Syntax
 
-Ethereum addresses were, historically, case-insensitive and normalized to use
-all-lowercase letters (`abcdef`) like most hexadecimal numeric types.  With the
-ratification of [EIP-55][], however, a particular normalization of lowercase- and
-uppercase- `abcdefABCDEF` characters was invented as an efficient form of
-checksum. See [EIP-55][] for specification.
+Ethereum addresses were, historically, case-insensitive and normalized to use all-lowercase letters (`abcdef`) like most hexadecimal numeric types.
+With the ratification of [EIP-55][], however, a particular normalization of lowercase- and uppercase- `abcdefABCDEF` characters was invented as an efficient form of checksum.
+Most implementations will still accept lowercase addresses but produce only checksum-case addresses, as many transaction-builders will validate not just against a regular expression but also against an EIP-55 checksum.
+Anywhere ["Postel's Law"][postel] can apply, implementers SHOULD produce checksum-case secure addresses (whether in CAIP-10 or native format), and SHOULD accept both checksum case and legacy lowercase addresses, except where the security concerns of the particular usecase outweigh interoperability.
```

Comment:
Nice still not making it mandatory (and this not a breaking change). But we still haven't guaranteed interoperability. Hopefully we can fix completely in the future

Reply:
well it's a final CAIP so literally we can't make a breaking normative change! what would fix completely look like? the whole point of this PR was to make status quo more explicit, any change to MUSTs and SHOULDs would have to be a new CAIP-number :D